### PR TITLE
Tables to profile

### PR DESCRIPTION
**Pull request by DorisSanchez-Velasquez**
FrontEnd : 

- Fixed a bug in the register route on the backend where image_url wasn't being updated properly when a user registers
- Added Ticket Statistics cards to the user profile page

### CODE REVIEW

**Comment by LifeBringer** [comment]
conflicts

**Review by LifeBringer — APPROVED** [review]
Good to go after resolving merge conflicts.

**Review by Rebecca-CYLi — APPROVED** [review]
LGTM

**Pull request merged**
